monitor/alerts/instance-delete-example/instance-delete-example.3.x.js

```diff
@@ -0,0 +1,18 @@
+// Download the Node helper library from twilio.com/docs/node/install
+// These are your accountSid and authToken from https://www.twilio.com/console
+const accountSid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX';
+const authToken = 'your_auth_token';
+
+const MonitorClient = require('twilio').MonitorClient;
+const client = new MonitorClient(accountSid, authToken);
+
+client.alerts('NO5a7a84730f529f0a76b3e30c01315d1a').delete(function(err, data) {
```

Comment:
this is the api you want to be using here: https://github.com/twilio/twilio-node/blob/next-gen/spec/integration/rest/monitor/v1/alert.spec.js#L80